src/backend/db/DB.ts

```diff
@@ -0,0 +1,15 @@
+import { Collection, Db, MongoClient } from "mongodb";
+import { GBScoutForm } from "../../types/GBScoutForm";
+
+const mongoURI = "mongodb://0.0.0.0:27017/GBScouting";
+
+const PromisedDatabase: Promise<Db> = MongoClient.connect(mongoURI).then(
+  (client) => client.db("GBScouting")
+);
+
+function collectionize<T extends {}>(name: string): Promise<Collection<T>> {
```

Comment:
This does not turn a string into a collection
```suggestion
function getCollection<T extends {}>(name: string): Promise<Collection<T>> {
```